### ISSUE TITLE
"Refresh All Tools" behaviour is odd if the quay.io token is deleted

### ISSUE THREAD

**Issue by denis-yuen** [ga4gh/dockstore#294]
Two issues

1. Published containers will become unpublished if the quay.io token is deleted
2. The token must be deleted on the quay.io side before we can re-link a quay.io account. Otherwise, attempting to re-link will fail and boot you back to the home page.

**Comment by denis-yuen** [comment]
Part 1) is fixed and deployed

**Comment by denis-yuen** [comment]
Part 2) is a duplicate of #304
